contracts/clients/package.json

```diff
@@ -9,21 +9,15 @@
   "license": "MIT",
   "private": false,
   "scripts": {
-    "build": "rm -rf dist && mkdir dist && cp -rH typechain dist/typechain && tsc",
+    "build": "rm -rf dist && mkdir dist && cp -rH typechain dist/typechain && find ./dist/typechain -type f \\! -name '*.d.ts' -name '*.ts' -delete && tsc",
     "watch": "tsc -w",
     "test": "mocha dist/**/*.test.js",
     "premerge": "yarn build && yarn test",
     "publish-experimental": "node scripts/showVersion.js >.version && npm version $(node scripts/showBaseVersion.js)-$(git rev-parse HEAD | head -c7) --allow-same-version && npm publish --tag experimental && npm version $(cat .version) && rm .version",
     "publish-experimental-dry-run": "node scripts/showVersion.js >.version && npm version $(node scripts/showBaseVersion.js)-$(git rev-parse HEAD | head -c7) --allow-same-version && npm publish --tag experimental --dry-run && npm version $(cat .version) && rm .version"
   },
   "dependencies": {
-    "@ethersproject/abi": "^5.4.0",
-    "@ethersproject/bignumber": "^5.4.1",
-    "@ethersproject/bytes": "^5.4.0",
-    "@ethersproject/keccak256": "^5.4.0",
-    "@ethersproject/sha2": "^5.4.0",
-    "@ethersproject/solidity": "^5.4.0",
```

Comment:
@voltrevo Given discussion today and what I saw in my tests this seemed safe to do. Let me know if I should revert.

Reply:
Nah, looks good. Nice one.